specification/resourcegraph/resource-manager/readme.md

```diff
@@ -40,6 +40,14 @@ semantic-validator: true
 model-validator: true
 message-format: json
 ```
+### Tag: package-preview-2020-04
+
+These settings apply only when `--tag=package-preview-2020-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-preview-2020-09'
+input-file:
+  - Microsoft.ResourceGraph/preview/2020-09-01-preview/resourcegraph.json
```

Comment:
This will cause other features to be not available in the SDK generated from this package.
I suggest refactoring the swagger into two parts with resourcechanges and history into the new swagger and refer appropriate versions in the package file.

Reply:
@chiragg4u  I created two specs resourcechanges and resourcegraph where latter contains /resources and /operations endpoints of the old version. Let me know if it looks good.